Source/Blazorise/Components/Dropdown/Dropdown.razor.cs

```diff
@@ -92,10 +93,9 @@ protected override void BuildClasses( ClassBuilder builder )
         {
             builder.Append( ClassProvider.Dropdown( IsDropdownSubmenu ) );
             builder.Append( ClassProvider.DropdownGroup(), IsGroup );
-            builder.Append( ClassProvider.DropdownShow(), Visible );
+            //builder.Append( ClassProvider.DropdownShow(), Visible );
```

Comment:
I think `DropdownShow` should still be here because we're doing this for BS only. Other providers should still work the same, right?

Reply:
Actually I'd like to do this for every provider. BS/BS5/Material are already pretty similar.
Bulma and ant-design aren't working right with responsive tables and hopefully this will help.

Reply:
OK, that makes sense.